Networking/Networking/Remote/BlazeRemote.swift

```diff
@@ -27,9 +52,68 @@ public final class BlazeRemote: Remote, BlazeRemoteProtocol {
         return try await enqueue(request, mapper: mapper)
     }
 
+    /// Fetches target languages for campaign creation.
+    ///
+    public func fetchTargetLanguages(for siteID: Int64) async throws -> [BlazeTargetLanguage] {
+        let path = BlazeTargetOption.languages.path(for: siteID)
+        let request = DotcomRequest(wordpressApiVersion: .wpcomMark2, method: .get, path: path)
```

Comment:
They seem to decide to use v1.1 instead.

peeHDf-2fw-p2#comment-1550

Reply:
I'm not sure the `v1.1` in the comment refers to the WPCom namespace, I think the API will still live under the same path as the previous one: `wpcom/v2/sites/{site-id}/wordads/dsp/api`, and this where the `v1.1` will be used instead of the current `v1` we have.

Reply:
I share the same assumption with Hicham. I left a comment in the `BlazeTargetOption .path(for: )` method to revise the paths when the API is finalized, so please consider this a temporary implementation for now.

Reply:
Oh I see, I was confused :D